src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,114 @@
-export const PeopleFilters = () => {
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import {
+  Dispatch,
+  SetStateAction,
+  useCallback,
+  useEffect,
+  useMemo,
+} from 'react';
+import { Person } from '../types';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+  setFilteredPeople: Dispatch<SetStateAction<Person[]>>;
```

Comment:
you don't need to filter the array in this component just handle the query params and filter the array in PeoplePage or PeopleTable